target/linux/ath79/dts/qca9558_dlink_dir-629-a1.dts

```diff
@@ -72,6 +72,11 @@
 	};
 };
 
+&eth1 {
+	nvmem-cells = <&macaddr_mfcdata_6a 0>;
+	nvmem-cell-names = "mac-address";
```

Comment:
This device uses the external switch chip connected to the `eth0`. It doesn't use eth1, so this patch won't work.

Reply:
oh so eth1 is not connected to the switch? that's unfortunate.

Reply:
Yes. QCA955x and QCA9563 don't have built-in switch.

Reply:
yeah, only two devices have ar8236.

Reply:
@Ansuel this switch looks similar to 8327, no?